### ISSUE TITLE
Coach reports date range should be inclusive

### ISSUE THREAD

**Issue by MCGallaspy** [learningequality/ka-lite#4337]
On `0.15.x`, created some student data, and then tried to view it on the coachreports as an admin. It seems that the date range is not inclusive in some cases, as student data from the same-day are not viewable:

![ss1](https://cloud.githubusercontent.com/assets/8888020/9620504/a198604a-50cf-11e5-8ebf-160924396049.png)

I tired several different start & end dates with student data from 9/1. In all cases I expect data to be show:

      Begin date      | End date  | Data shown?
---------|-----------|---------------------
 9/1 | 9/1 | No
9/1 | 9/2 | Yes
8/31 | 9/1 | No
8/31 | 9/2 | Yes

In other words, date ranges should be inclusive with both the end and begin dates.

**Comment by MCGallaspy** [comment]
Interestingly the default date range when I load the page is 8/25 to 9/1, and the tabular report is available. But if I click the "refresh" button next to the date range then the tabular report becomes unavailable.

**Comment by rtibbles** [comment]
Fixed in #4414
